notebooks/Revised_Example_LLH_setup_fixed_dirs.ipynb

```diff
@@ -0,0 +1,1528 @@
+       " 'Background+4U 1700-377+GRO J1655-40+GX 339-4+Sco X-1_Sco X-1_rate_8': 6.897789976904059e-05,\n",
+       " 'Signal_A': 1.0,\n",
+       " 'Signal_gamma': 0.5,\n",
+       " 'Signal_Epeak': 100.0}"
+      ]
+     },
+     "execution_count": 94,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pars_ = {}\n",
+    "pars_['Signal_theta'] = theta\n",
+    "pars_['Signal_phi'] = phi\n",
+    "for pname,val in list(bkg_params.items()):\n",
+    "    pars_[bkg_name+'_'+pname] = val\n",
+    "for pname,val in list(flux_params.items()):\n",
+    "    pars_['Signal_'+pname] = val\n",
+    "pars_\n",
+    "\n",
+    "# creating a dictionary for the data in the output of the previous cell and changing name so everything is prepended\n",
+    "# and has signal_A, gamma, Epeak."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the background and signal models assembled, a singular compound model can be created which combines the two models into one. **comp_mod** is a CompoundModel class object created using the **bkg_mod** and **sig_mod** variables."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 95,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "comp_mod = CompoundModel([bkg_mod, sig_mod])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Calculating and Minimizing Log Likelihood**\n",
+    "---\n",
+    "\n",
+    "The compound model can now be used to calculate the likelihood statistic. (display likelihood statistic equation). The parameters that minimize this statistic is pressumed to be the parameters that most likely corresponds to the observed data (bad phrasing).\n",
+    "\n",
+    "The calculation of the log likelihood is done through the LLH_webins class object, **sig_llh_obj**, which utilizes the event data (**ev_data0**), the lower and upper bounds of the energy bins (**ebins0** and **ebins1**), and the detector data (**bl_dmask**). **sig_llh_obj** makes use of the compound model by invoking the .set_model() function with **comp_mod** as an argument. When initializing **sig_llh_obj**, the argument has_err is set to True in order to utilize the error values set up in the compound model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 96,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sig_llh_obj = LLH_webins(ev_data0, ebins0, ebins1, bl_dmask, has_err=True)\n",
+    "sig_llh_obj.set_model(comp_mod)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The log likelihood minimization is handled by the NLLH_ScipyMinimize_Wjacob object, **sig_miner**, which is a wrapper of the scipy minimizer object. Using the .set_llh() function, it can act on **sig_llh_obj** to recalculate the log likelihood through variations of the parameters. "
```

Comment:
"recalculate the log likelihood through variations of the parameters"-->" to minimize the loglikelihood by varying the model parameters"